packages/xrpl/src/models/methods/accountChannels.ts

```diff
@@ -1,18 +1,111 @@
 import { BaseRequest, BaseResponse, LookupByLedgerRequest } from './baseMethod'
 
+/**
+ * Represents a payment channel.
+ */
 export interface Channel {
-  account: string
-  amount: string
-  balance: string
-  channel_id: string
-  destination_account: string
-  settle_delay: number
-  public_key?: string
-  public_key_hex?: string
-  expiration?: number
-  cancel_after?: number
-  source_tab?: number
-  destination_tag?: number
+  // Required Fields
+  /**
+   * The source address that owns this payment channel.
+   * This comes from the sending address of the transaction that created the channel.
+   */
+  Account: string
```

Comment:
The other files in the `methods/` folder use `snake_case` for the interface attributes of ledger objects. Do we have a naming convention?

This is a minor nit, not a binding suggestion

Reply:
This seems to be a bug - nice catch :) 

Judging from the response type on xrpl.org the fields are lowercase + snake_case in the response type.

Reply:
I was using the `PaymentChannel` transaction type, but should've been using a similar (but not identical) type that's embedded within the `account_channels` request.